### PR TITLE
Fix lab10+ requests to allow omitting the trailing slash in a URL

### PR DESCRIPTION
**Pull request by chrishtr**

### CODE REVIEW

File: src/lab10.py [hunk 1]
```diff
@@ -39,7 +39,10 @@ def request(url, top_level_url, payload=None):
     assert scheme in ["http", "https"], \
         "Unknown scheme {}".format(scheme)
 
+    if url.find("/") < 0:
```
Comment (pavpanchekha) [inline src/lab10.py:42]: I would use `"/" not in url`

**Review by pavpanchekha — COMMENTED** [review]
I'm concerned that this will not work when combined with the `resolve_url` function. Does this work when `resolve_url` receives a current URL without a terminating slash? If yes, good to merge (though see Python nitpick below, which appears in two places.)

File: src/lab10.py (hunk 1 above)
Comment (pavpanchekha) [reply src/lab10.py:42]: This appears in `lab6.py` as well.

Comment (chrishtr) [reply src/lab10.py]: Fixed

**Comment by chrishtr** [comment]
> Does this work when `resolve_url` receives a current URL without a terminating slash? 

Yes. That's why I had to edit resolve_url in lab6. Without that fix it wasn't working.

**Comment by chrishtr** [comment]
(There is also a test in this PR for that situation).

**Pull request merged**